spec/message.abnf

```diff
@@ -1,13 +1,16 @@
-message = [s] *(declaration [s]) body [s]
+message = pattern / complex-message
+
+complex-message = "{{" [s] *(declaration [s]) body [s] "}}"
 
 declaration = let s variable [s] "=" [s] expression
-body = pattern
+body = quoted-pattern
      / (selectors 1*([s] variant))
 
```

Comment:
Note that this makes the following a valid message:

```
{{{{I am a message}}}}
```

We could change `body` to be just:

```suggestion
body = selectors 1*([s] variant)
```